Description: Selects all usable external themes
 Kadu has a nice set of external (submitted by 3rd parties) themes and
 plugins.
 .
 This patch makes all usable external themes to be installed
Author: Patryk Cisek <patryk@debian.org>
---
diff --git a/varia/themes/emoticons/CMakeLists.txt b/varia/themes/emoticons/CMakeLists.txt
index d472623..20c3187 100644
--- a/varia/themes/emoticons/CMakeLists.txt
+++ b/varia/themes/emoticons/CMakeLists.txt
@@ -1,6 +1,7 @@
 set (EMOTICON_THEMES
 	penguins
 	tango
+        gg6_compatible
 )
 
 if (EMOTICON_THEMES)
diff --git a/varia/themes/sounds/CMakeLists.txt b/varia/themes/sounds/CMakeLists.txt
index 841c6ee..8beffb5 100644
--- a/varia/themes/sounds/CMakeLists.txt
+++ b/varia/themes/sounds/CMakeLists.txt
@@ -1,5 +1,11 @@
 set (SOUND_THEMES
 	default
+        bns
+        drums
+        florkus
+        michalsrodek
+        percussion
+        ultr
 )
 
 if (SOUND_THEMES)
